src/main/java/com/movie/store/controllers/ShoppingCartController.java

```diff
@@ -0,0 +1,47 @@
+package com.movie.store.controllers;
+
+import com.movie.store.model.MovieSession;
+import com.movie.store.model.User;
+import com.movie.store.model.dto.ShoppingCartResponseDto;
+import com.movie.store.service.interfaces.MovieSessionService;
+import com.movie.store.service.interfaces.ShoppingCartService;
+import com.movie.store.service.interfaces.UserService;
+import com.movie.store.service.mappers.ShoppingCartMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final ShoppingCartService shoppingCartService;
+    private final ShoppingCartMapper shoppingCartMapper;
+    private final UserService userService;
+    private final MovieSessionService movieSessionService;
+
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
```

Comment:
sometimes you use @Autowired before the constructor, sometimes not - let's be consistent